pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -77,13 +77,19 @@ template <class StateVectorT, class Derived> class MeasurementsBase {
     /**
      * @brief Randomly set the seed of the internal random generator
      *
-     * @param seed Seed
      */
     void setRandomSeed() {
         std::random_device rd;
         setSeed(rd());
     }
 
+    /**
+     * @brief Set the internal random generator to an already existing instance
+     *
+     * @param rng An already existing instance of a random number generator
+     */
+    void setRNG(std::mt19937 rng) { this->rng = rng; }
```

Comment:
This makes a copy of the object

Reply:
This method is no longer used now that we pass the seed number instead of the rng instance